sources/packages/backend/workflow/test/utils/assessment-consolidated-data-utils.ts

```diff
@@ -0,0 +1,109 @@
+import { OfferingIntensity } from "@sims/sims-db";
+import {
+  AssessmentConsolidatedData,
+  OfferingDeliveryOptions,
+  Provinces,
+  YesNoOptions,
+} from "../models/assessment.model";
+
+export function getFakeAssessmentConsolidatedData(
```

Comment:
I believe that this method should follow the name naming conventions (e.g. `createFakeSomething`) from the `libs/test-utils/factories` including the folder's name also (`test-utils` and `factories`).